guardian/atoms-rendering
# Add analytics for youtube video

## What does this change?
Adds support for `gaEventEmitter` and `ophanEventEmitter` in YoutubeAtom. In order to do so a lot of logic was lifted from YoutubeOverlay component and we need to now track when a video is played and its progress.

Events that we want to track:
- On video is played
- Reached 25% 
- Reached 50%
- Reached 75%
- Finished video

## How to test
Run `yarn storybook` look at the different youtube stories. You should be able to see when an analytics event emiter is called in the console.

## How can we measure success?
Make sure only one event get fired for each state of the Youtube video
